File: strings/is-the-string-empty.py
def is_empty(s):
	return len(s) == 0

is_empty("") # True
is_empty(" ") # False
is_empty("            ") # False
is_empty("38215") # False
is_empty("afjabsdf") # False
is_empty("!?@&") # False

File: strings/is-the-string-empty.cpp
#include <string>

bool isEmpty(std::string str) {
	return str.empty();
}

Describe(empty_string_validation)
{
	It(should_return_true)
	{
		Assert::That(isEmpty(""), Equals(true));  
	}
	It(should_return_false_even_for_strings_with_only_white_spaces)
	{
		Assert::That(isEmpty(" "), Equals(false));
		Assert::That(isEmpty("            "), Equals(false));
	}
	It(should_return_false_for_all_other_strings)
	{
		Assert::That(isEmpty("38215"), Equals(false));
		Assert::That(isEmpty("afjabsdf"), Equals(false));		
		Assert::That(isEmpty("!?@&"), Equals(false));	
	}
};